### PR TITLE
fix(components): text-area resize (VIV-1004)

### PR DESCRIPTION
**Pull request by olaf-k**

### CODE REVIEW

File: libs/components/src/lib/text-area/README.md [hunk 1]
```diff
@@ -49,6 +49,20 @@ Set the `value` attribute to set the default value for the text area. Setting th
 <vwc-text-area value="Default Value"></vwc-text-area>
 ```
 
+### Resize
+
+- Type: `none` | `vertical` | `horizontal` | `both`
+- Default: `none`
```
Comment (rachelbt) [inline libs/components/src/lib/text-area/README.md:55]: why none? in native text-area its initially auto which is: both
image

Comment (olaf-k) [reply libs/components/src/lib/text-area/README.md:55]: true. I chose to keep Fast's and Vivid 2's default behavior to minimize code impact.

Comment (rachelbt) [reply libs/components/src/lib/text-area/README.md:55]: I guess it make sense :)

**Review by rachelbt — COMMENTED** [review]
sorry for being the one with tedious comments - but why is the border-bottom of FF png's are missing?

**Comment by olaf-k** [comment]
> sorry for being the one with tedious comments - but why is the border-bottom of FF png's are missing?

no worries, it's a fair question. removing the 5px margin for each sample's div display the page properly but samples are then stacked with Chrome/WebKit. increasing it seems to work for all of them.
I have no idea why the default margin we use results in this though :/

**Review by rachelbt — APPROVED** [review]

**Pull request merged**